image-inspector.go

```diff
@@ -249,6 +253,19 @@ func main() {
 
 	supportedVersions := APIVersions{Versions: []string{VERSION_TAG}}
 
+	var arf_body []byte
+	if *oscap {
+		log.Printf("OpenScap scanning %s", *dstpath)
```

Comment:
@enoodle the official name with correct case is `OpenSCAP`. Same for all the other entries below.